src/app/app.module.ts

```diff
@@ -2,13 +2,28 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
-
+import { HeaderComponent } from './header/header.component';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
+import {AppRoutingModule} from "./app-routing.module";
+import { TrainingComponent } from './training/training.component';
+import { PricingComponent } from './pricing/pricing.component';
+import { FeaturesComponent } from './features/features.component';
+import {FormsModule} from "@angular/forms";
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    HeaderComponent,
+    LoginComponent,
+    RegisterComponent,
+    TrainingComponent,
+    PricingComponent,
+    FeaturesComponent
   ],
   imports: [
-    BrowserModule
+    BrowserModule,
+    FormsModule,
```

Comment:
Browser, and Forms module move to shared module

Reply:
done